addmanager.go

```diff
@@ -72,7 +85,10 @@ func addProjectHandler(projName string, projType string, init bool) error {
 		return errors.New("Unsupported project type specified")
 	}
 	if init {
-		createAndInit(projName)
+		err := createAndInit(projName)
+		if err != nil {
+			return err
+		}
```

Comment:
Can this just be `return err`? Seems like if `err == nil` then you just end the case and return `nil` anyway.

Reply:
Yes, that should be changed. Let me do that...